src/libasr/pass/global_stmts_program.cpp

```diff
@@ -18,15 +18,14 @@ using LFortran::ASRUtils::EXPR;
  *
  */
 void pass_wrap_global_stmts_into_program(Allocator &al,
-            ASR::TranslationUnit_t &unit) {
-    std::string fn_name_s = "_lfortran_main_program";
+            ASR::TranslationUnit_t &unit, std::string name) {
```

Comment:
Let's make the argument more descriptive, such as:
```suggestion
            ASR::TranslationUnit_t &unit, std::string program_fn_name) {
```

Reply:
Also, pass it with `&`.

Reply:
And `const`.